### ISSUE TITLE
Check JS cache busting

### ISSUE THREAD

**Issue by georgjaehnig** [trovu/trovu#409]
A recent `npm update` was available only after deep reload (Ctrl-Shift-R).

Check if JS cache busting is needed, e.g. with the commit hash in `index.123abc78.js`.
<br/>
<hr/>

<details><summary>This repo is using Opire - what does it mean? 👇</summary><br/>💸Everyone can add rewards for this issue commenting <code>/reward 100</code> (replace <code>100</code> with the amount)<br/>💪 If someone starts working on this issue to earn the rewards, they can comment <code>/try</code> to let everyone know!<br/>🙌 And when they open the PR, they can comment <code>/claim #409</code> either in the PR description or in a PR's comment<br/><br/>👀 Also, everyone can tip any user commenting <code>/tip 20 @georgjaehnig</code> (replace <code>20</code> with the amount, and <code>@georgjaehnig</code> with the user to tip)<br/><br/>If you want to learn more, go to our <a href="https://docs.opire.dev">documentation</a></details>

**Comment by erik55** [comment]
How to do ctrl+shift+r on an android phone? Do I have to tell all my not so techt-savvy friends now to do this? trovu stopped working for me yesterday (or the day before). Is there no other fix?

**Comment by georgjaehnig** [comment]
I'm sorry to hear that. What exactly do you experience with "stopped working"? The original issue was only about the non-working settings icon (#407).

Basically, it's about the JS files. Another option to make it run could be to open

- https://trovu.net/index.js
- https://trovu.net/process.js

and normally reload them.

**Comment by georgjaehnig** [comment]
Ah, I just can reproduce this on Android, too. Looking into this.

Edit: So the issue appears on Chrome or Brave for Android, but not on Firefox for Android.

Reloading https://trovu.net/index.js / https://trovu.net/process.js does not help.

Not sure if this is related to cache-busting. Opening a new ticket #410

**Comment by georgjaehnig** [comment]
Done with #410
